etc/desi.json

```diff
@@ -6,6 +6,23 @@
     },
     "datachallenge":{
         "exclude":[],
+        "dc17a-twopct":{
+            "dc17a-twopct/[README|twopct.ecsv|dc17a-lite.tar.gz]":"dc17a-twopct/toplevel.tar",
+            "dc17a-twopct/batch/.*":"dc17a-twopct/batch.tar",
```

Comment:
The regex lines (the keys in these dictionaries) should end with `$`.  See other examples in the same file.